src/test/java/org/opentripplanner/middleware/OtpMiddlewareTest.java

```diff
@@ -29,7 +32,20 @@ public static void setUp() throws RuntimeException, IOException {
         LOG.info("OtpMiddlewareTest setup");
 
         LOG.info("Starting server");
-        otpMiddleware.main(new String[]{"configurations/test/env.yml"});
+        OtpMiddlewareMain.inTestEnvironment = true;
+        // If in the e2e environment, use the secret env.yml file to start the server.
+        // TODO: When ran on Travis CI, this file will automatically be setup.
+        String[] args = getBooleanEnvVar("RUN_E2E")
```

Comment:
Given how much `getBooleanEnvVar("RUN_E2E")` is used, I recommend making it a read-only field for this class.

Reply:
Punting to @landonreed https://github.com/ibi-group/otp-middleware/pull/69/files#r483890844.